### ISSUE TITLE
Crashes with a Nullpointer in SmoochService

### ISSUE THREAD

**Issue by ykostova** [zendesk/sunshine-conversations-android#61]
```
Fatal Exception: java.lang.NullPointerException: Attempt to invoke virtual method 'void io.smooch.core.service.SmoochService.c(java.lang.Runnable)' on a null object reference
       at io.smooch.core.b$2.run(Unknown Source)
       at io.smooch.core.service.SmoochService.A(Unknown Source)
       at io.smooch.core.service.SmoochService.a(Unknown Source)
       at io.smooch.core.service.SmoochService.a(Unknown Source)
       at io.smooch.core.service.SmoochService$19.a(Unknown Source)
       at io.smooch.core.service.SmoochService$19.a(Unknown Source)
       at io.smooch.core.b.f$1.a(Unknown Source)
       at io.smooch.core.b.a.b$1$2.run(Unknown Source)
       at android.os.Handler.handleCallback(Handler.java:739)
       at android.os.Handler.dispatchMessage(Handler.java:95)
       at android.os.Looper.loop(Looper.java:135)
       at android.app.ActivityThread.main(ActivityThread.java:5910)
       at java.lang.reflect.Method.invoke(Method.java)
       at java.lang.reflect.Method.invoke(Method.java:372)
       at com.android.internal.os.ZygoteInit$MethodAndArgsCaller.run(ZygoteInit.java:1405)
       at com.android.internal.os.ZygoteInit.main(ZygoteInit.java:1200)
```

**Comment by ykostova** [comment]
##Device stats for the observed crashes: 

<img width="1008" alt="screen shot 2016-11-29 at 12 32 07" src="https://cloud.githubusercontent.com/assets/2377138/20709936/eb7a9dec-b62f-11e6-82af-db214d601abf.png">

**Comment by wmora** [comment]
thanks for reporting, we'll take a look at it

**Comment by DataWearsAHood** [comment]
Hello!

Looks like a few nullpointer crashes have been fixed since this issue was first reported (v5.7.1, v5.15.0 v7.0.2).
Please reopen this issue if you are still experiencing this crash with the latest release.
